algorithms/ar-isrt/typescript/insertion_sort.ts

```diff
@@ -0,0 +1,32 @@
+const swap = (arr: Array<number>, a: number, b: number)=>{
+	let temp = arr[a];
+	arr[a] = arr[b];
+	arr[b] = temp;
+}
+
+const insertionsort = (values: Array<number>)=>{
+
+	for (let i = 1; i < values.length; i++) 
+    { 
+        let j = i; 
+        
+        while (j > 0 && values[j] <  values[j -1]) 
+        { 
+           	 swap(values, j, j - 1);
+        	
```

Comment:
Codacy found an issue: [Mixed spaces and tabs.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6355990)